cmd/build/helmify/main.go

```diff
@@ -98,7 +98,7 @@ func (ks *kindSet) Write() error {
 			}
 
 			if name == "gatekeeper-mutating-webhook-configuration" {
-				obj = "{{- if .Values.experimentalEnableMutation }}\n" + obj + "{{- end }}\n"
+				obj = "{{- if .Values.experimentalEnableMutation }}\n" + strings.Replace(obj, "failurePolicy: Ignore", "failurePolicy: {{ .Values.mutatingWebhookFailurePolicy }}", 1) + "{{- end }}\n"
```

Comment:
Could we add a kustomize entry for the MutatingWebhookConfiguration like we do for the ValidatingWebhookConfiguration?

https://github.com/open-policy-agent/gatekeeper/blob/0f910c14e040f32b38860e6e98675a1b96b56bf2/cmd/build/helmify/kustomize-for-helm.yaml#L173-L203

That would prevent this change from breaking if `Ignore` stops being the static failure policy.

Reply:
@maxsmythe Done. And I have added timeout configuration too.

However, tests are now failing. Any idea?